code-challenges.test.js

```diff
@@ -42,12 +55,33 @@ var cubeAndSum1 = [2, 3, 4]
 var cubeAndSum2 = [0, 5, 10]
 // Expected output: 1125
 
+describe ("when 'cubedSum' is called", () => {
+    let cubeAndSum1 = [2, 3, 4]
+    let cube1 = 99
+    let cubeAndSum2 = [0, 5, 10]
+    let cube2 = 1125
+    it ("sums and cubes the numbers", () => {
+        expect (cubedSum(cubeAndSum1)).toEqual(cube1)
+    })
+    it ("subs and cubes the numbers'", () => {
+        expect (cubedSum(cubeAndSum2)).toEqual(cube2)
+    })
+})
 
 
 
 
-// b) Create the function that makes the test pass.
+// // b) Create the function that makes the test pass.
+
+// we want to sum the numbers over the array and multiply those summed values by three
 
+const cubedSum = (array) => {
+    let currentSum = 0
+    for (let i=0; i < array.length; i++) {
```

Comment:
very nice! this would also be a good opportunity to look for built-in methods that would allow you to do almost the same thing